docs/content.zh/docs/deployment/speculative_execution.md

```diff
@@ -40,10 +40,6 @@ under the License.
 ## 使用方法
```

Comment:
I noticed an existing problem that on the navigation bar of zh pages, it's "Speculative Execution" instead of "预测执行".
Could you add a hotfix to fix it?

Reply:
I checked the master branch. The issue has been resolved through https://github.com/apache/flink/commit/cefb3339514a3fea1ad44eaaf5ad8be9756d9d46.